runelite-client/src/main/java/net/runelite/client/plugins/inferno/InfernoJad.java

```diff
@@ -24,30 +24,74 @@
  */
 package net.runelite.client.plugins.inferno;
 
+import lombok.AccessLevel;
+import lombok.Getter;
 import net.runelite.api.AnimationID;
+import net.runelite.api.NPC;
 import net.runelite.api.Prayer;
 
-public enum InfernoJadAttack
+public class InfernoJad
```

Comment:
`@Getter(AccessLevel.PACKAGE)` above a class does it all for you, no need to individually mark them.